app/models/events/review.rb

```diff
@@ -46,5 +46,15 @@ class Review < ApplicationRecord
 
     validates :state, inclusion: { in: states.keys }
     validates :github_id, :opened_at, presence: true
+
+    after_create :build_review_turnaround
```

Comment:
maybe you can add a test for this business logic?

Reply:
Yes! I forgot